cob_sound/action/Say.action

```diff
@@ -1,5 +1,5 @@
 # goal
-std_msgs/String text
+string text
```

Comment:
as now none of the messages requires `std_msgs` in its definition anymore, please remove `std_msgs` from https://github.com/ipa-nhg/cob_driver/blob/95181a8b5b666d5e76c77464a3513465c5d79cd1/cob_sound/CMakeLists.txt#L13